develop/tutorials/articles/280-workflow/01-creating-workflow-definitions/03-notifications.markdown

```diff
@@ -0,0 +1,78 @@
+# Workflow Notifications
+
+While an asset is in a workflow, it's important that relevant Users are notified
+about certain events, like when a review task is completed. Any workflow node
+that can take an `<actions>` element can have notifications.
```

Comment:
Is this information based on the DTD? Honestly, I was unsure about this statement. 😁

Reply:
Yes, it was base don my understanding of the DTD. to confirm I just went and checked in Kaleo Designer. Conditions, States, Forks, Joins, and Tasks can all have notifications and actions.I think this is accurate then, right?

Reply:
Yeah. Once you've checked the DTD it's fine, because the Kaleo Designer sometimes can have an issue, like the one which has added the `<execution-type>` on the `<timer-action>`.